user_docs/en/changes.md

```diff
@@ -51,6 +52,9 @@ Add-ons will need to be re-tested and have their manifest updated.
   * A `toggleBooleanValue` helper function has been added to `globalCommands`.
   It can be used in scripts to report the result when a boolean is toggled in `config.conf`
 * Removed the requirement to indent function parameter lists by two tabs from NVDA's Coding Standards, to be compatible with modern automatic linting. (#17126, XLTechie)
+* Correct the internal IDs of the braille input gesture objects for Seika Notetaker. (#17047, @school510587)
+  * Parameter space of class InputGesture constructor becomes an integer between 0 and 3.
+  * Gestures involving br(seikantk):BACKSPACE and/or br(seikantk):SPACE can be correctly linked to command scripts.
```

Comment:
Sorry, this should be under API breaking changes.
The issue is that you are breaking the API, we don't need a general description of the code changes here.
Please read [this doc](https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/deprecations.md) for more information.

The message should be something like


```py
* The `space` keyword argument for `brailleDisplayDrivers.seikantk.BrailleDisplayDriver` now expects an `int` not a `bool`. (#17047)
```